rotkehlchen/tests/unit/test_optimism_transactions.py

```diff
@@ -49,3 +51,9 @@ def assert_tx_okay(transactions, should_have_l1):
             has_premium=True,
         )
         assert_tx_okay(transactions, should_have_l1=False)
+
+    # check that the get_or_create_transaction method makes sure that the requirements for an
+    # optimism transaction (l1_fee existing in the database) are met before returning it.
+    with optimism_transactions.database.conn.read_ctx() as cursor:
+        tx, _ = dbevmtx.get_or_create_transaction(cursor, optimism_inquirer, tx_hash, string_to_evm_address(to_address))  # noqa: E501
```

Comment:
hoooo I think this will go boom boom. Moved `get_or_create_transaction` to `EVMTransactions`. Linting passed which means linting skips this test file. Will need to fix.